contracts/hts-precompile/IHederaTokenService.sol

```diff
@@ -157,6 +157,18 @@ interface IHederaTokenService {
     struct TokenInfo {
         /// The hedera token;
         HederaToken hedera;
+        
+        /// The number of tokens (fungible) or serials (non-fungible) of the token
```

Comment:
Does this align with the expected order in the HTS Precompile logic in services?  The ABI does not do symbols but matches strictly by ordinal positions.

Reply:
Asked the same question [above](https://github.com/hashgraph/hedera-smart-contracts/pull/57#discussion_r969559646) and @georgi-l95  claims it does